db/models.py

```diff
@@ -1 +1,38 @@
 from django.db import models
+
+# Model for Race
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self):
+        return self.name
+
+# Model for Skill
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE, related_name='skills')
+
+    def __str__(self):
+        return self.name
+
+# Model for Guild
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
+
+    def __str__(self):
+        return self.name
+
+# Model for Player
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255, null=False)
+    bio = models.CharField(max_length=255)
```

Comment:
Consider adding `blank=True` to the `bio` field in the `Player` model if you want to allow it to be optional in forms. Currently, it is required in forms but not in the database.